File: Array/fanbei/0626/java/Solution06_26.java
package com.example.lsn37_dagger2_subcomponent;

public class Solution06_26 {

    static class ListNode{
        int val;
        ListNode next;

        ListNode(int x){
            val = x;
            next = null;
        }
    }

    class Solution{
        public void deleteNode(ListNode node){
            node.val = node.next.val;
            node.next = node.next.next;
        }
    }

}

File: Array/fanbei/0626/c/solution06_26.c
#include <stdlib.h>
#include <stdio.h>

/**
 * 删除链表中的节点
 */
struct ListNode{
    int val;
    struct ListNode* next;
};

void deleteNode(struct  ListNode* node){
    node->val = node->next->val;
    node->next = node->next->next;
}

int main(){}


